src/components/endpointCreation/NewCaptureModal.tsx

```diff
@@ -253,19 +274,23 @@ function NewCaptureModal(
         } else if (schema && schema.connectionSpecification !== null) {
             return (
                 <ErrorBoundary>
-                    {schema.documentationUrl ? (
-                        <AppBar
-                            position="relative"
-                            elevation={0}
-                            color="default"
+                    <AppBar position="relative" elevation={0} color="default">
+                        <Toolbar
+                            variant="dense"
+                            sx={{
+                                justifyContent: 'space-between',
+                            }}
                         >
-                            <Toolbar variant="dense">
+                            <Typography variant="h5" color="initial">
+                                {image}
+                            </Typography>
+                            {schema.documentationUrl ? (
                                 <ExternalLink link={schema.documentationUrl}>
                                     Connector Docs
                                 </ExternalLink>
-                            </Toolbar>
-                        </AppBar>
-                    ) : null}
+                            ) : null}
+                        </Toolbar>
+                    </AppBar>
```

Comment:
Adding the full image path to the toolbar so always showing the toolbar for now.